WordPress/Jetpack/Resources/release_notes.txt

```diff
@@ -1,12 +1,19 @@
-What time is it? Our brand-new time zone selection screen knows. Get a suggested time zone based on your device and improve search while you’re at it.
+* [*] Signup: Fixed bug where username selection screen could be pushed twice. [#17624]
+* [**] Threaded comments: comments can now be moderated via a drop-down menu on each comment. [#17758]
+* [**] Accessibility: VoiceOver and Dynamic Type improvements on Activity Log and Schedule Post calendars [#17756, #17761, #17780]
+* [*] Weekly Roundup: Fix a crash which was preventing weekly roundup notifications from appearing [#17765]
+* [*] Self-hosted login: Improved error messages. [#17724]
```

Comment:
Is that applicable to Jetpack, given it's about Self-hosted? \cc @leandroalonso

Reply:
No, not applicable. Self-hosted is not supported in the Jetpack app.